1-estrategia/README.md

```diff
@@ -2,22 +2,129 @@
+
+__Dónde__: 
+
+Pais Vasco. 
+
+__Para que__: 
+
+Facilitar la integración de los inmigrantes en la comunidad Vasca, brindarles esperanza y que puedan superar la crisis en la que se ven sumergidos.
+
+
 ### Criterios de diseño
 
+Aupa tiene como fin comunicar e informar a los migrantes del País Vasco, por ende los criterios de diseño los definimos de la siguiente manera: 
+
+* Accesibilidad: El sitio web está pensado para obtener acceso multipantalla, es decir, nuestros usuarios pueden ingresar al sitio web desde la computadora, la tablet o el teléfono móvil. 
+
+* Findability:  En nuestro sitio web, la intuitividad es muy importante, las pestañas deben ser muy claras y con una organización fácil para los inmigrantes. 
+
+* Funcionalidad: se refiere a que todo lo disponible en la web funcione correctamente. 
+
+* Utilidad: hace referencia al uso que el usuario pueda dar a la web. Es decir, que la información y la manera de ordenarla sea correcta para cumplir con las necesidades del público objetivo.
+
+* Estética: Esta cualidad busca una apariencia amigable para el usuario. La elección de la tipografía es un ejemplo de criterio influenciado por este factor. → Para nuestro sitio web, la simplicidad es un factor muy importante porque no todos nuestros usuarios dominan la lengua hispana, por esta razón no integraremos textos largos, a su vez, complementaremos con imágenes y multimedia para enriquecer nuestro sitio y ayudar a que se pueda encontrar de una manera más rápida lo que buscan. Es decir, el diseño de la plataforma está pensado en simplicidad desde las pestañas del menú hasta el contenido informativo.
+
+* Intuitividad: Otro de los propósitos de la plataforma web es que el usuario pueda vivir una experiencia agradable y sin frustraciones, esto se logra mediante técnicas de navegación que sean claras, reduciendo la ambigüedad en la interpretación, es decir, si lo que busca es información sobre la documentación, encontrará una pestaña que diga “Trámite de documentos”, y que al clickear salga efectivamente la información sobre dónde debe ir, qué requisitos cumplimentar, las entidades a las cuales puede contactar. Un error que le genere frustración al usuario sería que al momento de clickear en la pestaña en lugar de encontrar esa información encuentre micro historias de las experiencias de otros migrantes en la ciudad.
+
+* Credibilidad: La credibilidad en nuestro sitio se debe sostener en parte a partir del trabajo ético, transparente, moral y profesional de todos los que forman parte de este sitio web, la otra parte, vendrá dado gracias a las alianzas con instituciones del orden público y reconocidas ONGs. Presentaremos de manera visible, una sección dedicada a los Partners, que serán justamente estas instituciones y organizaciones con quienes trabajamos conjuntamente. Todo esto ayudará en el tiempo a fortalecer la confianza, fiabilidad y credibilidad en nuestro sitio web.
+
 ### Indicadores de éxito
```

Comment:
Todos los indicadores que planteáis requieren preguntar al usuario. ¿Creéis que estará dispuesto a respodner a tantas preguntas? ¿Se os ocurren otros métodos (observación, analítica web) para poder obtener informaciones parecidas o extraer conclusiones similares?